core/src/main/java/hudson/widgets/HistoryWidget.java

```diff
@@ -214,4 +262,16 @@ public void setNextBuildNumberToFetch(String nextBuildNumberToFetch) {
         boolean isBuilding(T record);
         String getNextKey(String key);
     }
+
+    private Long getPagingParam(StaplerRequest currentRequest, String name) {
```

Comment:
I don't really think it needs it since it's an internal method, but will add it.